lib/src/rules/do_not_create_setter_without_getter.dart

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library linter.src.rules.avoid_setter_without_getter;
```

Comment:
nit: the library name didn't get updated. I'm surprised there was no error from importing multiple libraries with the same name. (By the way, library names are not needed, so one way to fix this would be to remove the library directive.)